# ==============================================================================
# FIXING COMMIT DIFF
# ==============================================================================
# File: netwerk/protocol/http/TLSTransportLayer.h
# Commit: 970422f6006c
# Full Hash: 970422f6006c712e04ad97a1712a217677a7a01a
# Author: Kershaw Chang <kershaw@mozilla.com>
# Date: 2023-08-17 08:54:42
# Description:
#   Bug 1848999 - Always release TLSTransportLayer on socket thread, r=necko-reviewers,valentin
#   
#   Differential Revision: https://phabricator.services.mozilla.com/D186388
# ==============================================================================

diff -r 44ba8ebd6477 -r 970422f6006c netwerk/protocol/http/TLSTransportLayer.h
--- a/netwerk/protocol/http/TLSTransportLayer.h	Thu Aug 17 00:55:25 2023 +0000
+++ b/netwerk/protocol/http/TLSTransportLayer.h	Thu Aug 17 01:59:11 2023 +0000
@@ -129,6 +129,7 @@
   };
 
   virtual ~TLSTransportLayer();
+  bool DispatchRelease();
 
   nsISocketTransport* Transport() { return mSocketTransport; }
 
